.travis.yml

```diff
@@ -2,29 +2,39 @@ language: php
 
 sudo: false
 
-php:
-  - 7.0
-  - 7.1
-  - 7.2
-
 matrix:
   include:
     - php: 7.0
       env: INFECTION=1
     - php: 7.1
       env: INFECTION=1
+    - php: 7.1
+      env: INFECTION=1 SYMFONY_VERSION="^4.0"
     - php: 7.2
       env: INFECTION=1
+    - php: 7.2
+      env: INFECTION=1 SYMFONY_VERSION="^4.0"
 
 cache:
   directories:
-  - $HOME/.composer/cache
+    - $HOME/.composer/cache
 
 before_install:
   - composer selfupdate
+  - if [ "$TRAVIS_PHP_VERSION" != "7.0" ]; then sed -i 's/"config":\ {"platform":\ {"php":\ "7.0.8"}},//' composer.json; fi
```

Comment:
could you please use what @borNfreee suggested here https://github.com/infection/infection/pull/71#issuecomment-352874969 instead of using `sed` ?

Reply:
Done.